Client/Frontend/Browser/Browser.swift

```diff
@@ -78,10 +79,23 @@ class Browser: NSObject {
             webView.navigationDelegate = navigationDelegate
             helperManager = HelperManager(webView: webView)
 
-            if let request = lastRequest {
+            if let sessionData = self.sessionData {
+                var jsonDict = [String: AnyObject]()
```

Comment:
Move this variable down to where you first use `jsonDict`. Also, `var` -> `let`.